tests/foreman/ui_airgun/test_settings.py

```diff
@@ -172,7 +174,7 @@ def test_positive_update_restrict_composite_view(session, set_original_property_
 
 
 @tier1
```

Comment:
This is still a tier1 test , please check

Reply:
Yes, it is a tier1 test. what do you recommend?

Reply:
under UI , we do not create tier1 tests and covering more under one `API/CLI` test only.
Do you think same type of functionality covered under exists `tier2/3` tests ? if yes then good to remove this one and if no then would say to cover under any `tier2/3` for same.

Reply:
as you can see, we have zero tier1 in web ui, because it is a requirement to new framework.

Reply:
OK I will remove this test, as per your comments mentioned above.